docs/data/toolpad/concepts/custom-components.md

```diff
@@ -2,11 +2,22 @@
 
 <p class="description">You can bring your own custom components inside the Toolpad editor.</p>
 
-## `createComponent`
+The steps below explain how to create and use a custom component in Toolpad.
 
-Toolpad exposes a `createComponent` API to bring in custom components to the Toolpad component library.
+## Create component
 
-For example, in `toolpad/components/HelloWorld.tsx`:
+1. Click **Create** in the custom component section of the Component library.
+2. Name your component.
+3. Open the snackbar prompt. It will open your code editor. In case it doesn't open, check [troubleshoot missing editor.](https://mui.com/toolpad/how-to-guides/editor-path/)
```

Comment:
Let's add the note of the failing editor separately? Also punctuation should go outside of the link and we should let the markdown parser auto-number list items (all of them should start with `1.`)

```suggestion
1. In the snackbar that appears, click the **Open** button. Your code editor will open with the newly created component. 

::info
In case it doesn't open, check [troubleshoot missing editor].(https://mui.com/toolpad/how-to-guides/editor-path/)
::
```